### DISCUSSION TITLE
How to disable "go to definition"?

### DISCUSSION THREAD

**Discussion by gaoqiangks** [James-Yu/LaTeX-Workshop#3563] [Q&A] [answered]
I know vscode  has a "go to definition" feature, which is very useful in other programming language. However, I never use it with latex language.   It will pop up a menu like "Could not find definition for \lambda" when I accidentally pressed the wrong key, which may occur more frequently in VIM mode.  Is there any way to disable this feature?

**Comment by James-Yu** [comment]
Correct me if I am wrong, but I don't think this is an extension-side issue. You may look for vscode configurations to see if you may disable the alert particular for languageID `latex`. I am not aware of such a config though.

**Comment by jlelong** [comment, marked as answer]
You can override a specific key binding rule with an empty action, see https://code.visualstudio.com/docs/getstarted/keybindings#_removing-a-specific-key-binding-rule. You can use a when clause to only do this in LaTeX mode `"when": "editorLangId == 'latex'"`.
